spec/p2p/implementation/switch.md

```diff
@@ -50,11 +50,11 @@ The `DialPeersAsync` method receives a list of peer addresses (strings)
 and dials all of them in parallel.
 It is invoked in two situations:
 
-- In the [setup](https://github.com/cometbft/cometbft/blob/29c5a062d23aaef653f11195db55c45cd9e02715/node/node.go#L985) of a node, to establish connections with every configured
-  persistent peer
+- In the [setup](https://github.com/cometbft/cometbft/blob/v0.34.x/node/node.go#L987)
```

Comment:
v0.34.x could still be updated and these links be broken. What is the guideline here? Should we incur in the risk of broken links or point to specific patch versions?

Reply:
So, the `implementation/README.md` specifies that this documentation is based on `v0.34.x` branch. I've tried to check whether replacing links to the `main` branch could to the work, but it doesn't. So I would rather maintain references to `v0.34.x` here, until we find a way to describe the implementation on all branches consistently.